### ISSUE TITLE
The experimental results could not be reproduced

### ISSUE THREAD

**Issue by chenqi008** [hangligit/InterpretDiffusion#3]
Thank you so much for releasing the codes. However, when I try reproducing the results based on the given instructions, the results seem to be a large difference from the values in the paper. For example,

profession | a_woman_bias_cnt | a_man_bias_cnt | deviation_ratio
-- | -- | -- | --
Analyst | 257 | 243 | 0.028
CEO | 201 | 299 | 0.196
Laborer | 167 | 333 | 0.332
Secretary | 445 | 55 | 0.78
Teacher | 406 | 94 | 0.624

The instructions I use are:
python test.py --train_data_dir datasets/person/ --output_dir exps/example_person --evaluation_type winobias --num_test_samples 50 --template_key 0 --concept 'woman' 'man' --clip_attributes 'a woman' 'a man'

Thanks again for such a promising work and code.

**Comment by ShristiDasBiswas** [comment]
were you able to reproduce the results? what were the required changes?

**Comment by chenqi008** [comment]
No, I can't. The results are still lower than that in the paper.

**Comment by chenqi008** [comment]
> were you able to reproduce the results? what were the required changes?

No, I can't. The results are still lower than that in the paper.
